src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendConsentOrderAboutToSubmitHandler.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.controllers.GenericAboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.CaseFlagsService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.ConsentOrderService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.LATEST_CONSENT_ORDER;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AmendConsentOrderAboutToSubmitHandler implements CallbackHandler<Map<String, Object>> {
```

Comment:
shouldn't this be using FinremCaseData rather than Map?

Reply:
Caseflag service need to be change- based on pojo. As this is incident so want to do quickly.